openbas-front/src/admin/components/settings/Parameters.tsx

```diff
@@ -379,6 +379,13 @@ const Parameters = () => {
             <ListItemText primary={t('RabbitMQ')} />
             <ItemBoolean status={null} variant="large" neutralLabel={settings?.rabbitmq_version} />
           </ListItem>
+          {settings.analytics_engine_type !== undefined
```

Comment:
```suggestion
          {settings.analytics_engine_type
```

because I am facing this issue
Error.js:56 Error: [@formatjs/intl] An `id` must be provided to format a message. You can either:
1. Configure your build toolchain with [babel-plugin-formatjs](https://formatjs.github.io/docs/tooling/babel-plugin)
or [@formatjs/ts-transformer](https://formatjs.github.io/docs/tooling/ts-transformer) OR
2. Configure your `eslint` config to include [eslint-plugin-formatjs](https://formatjs.github.io/docs/tooling/linter#enforce-id)
to autofix this issue
    at translate (i18n.tsx:184:59)
    at Parameters (Parameters.tsx:385:40)


The above error occurred in the <Parameters> component.

React will try to recreate this component tree from scratch using the error boundary you provided, ErrorBoundaryComponent.

Reply:
Good call !